app/views/devise/registrations/edit.html.erb

```diff
@@ -3,6 +3,10 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
+  <div class=”field”> (Upload your new profile picture below!)
```

Comment:
@NitinNROR did you copy paste this line? I see that the double quotes does not look like keyboard double quotes... can you pls check if this code is working...?

Reply:
yes... I did copy this line... most of the code I type in, but some I did copy then edit it. but this one was exact so I just put in...